scripts/build.sh

```diff
@@ -188,7 +188,8 @@ GIT_BRANCH=$(git branch | grep \* | cut -d' ' -f2)
 VERSION_STRING="$GIT_BRANCH:$VERSION"
 GIT_HASH=$(git describe --always --abbrev=50 --dirty --long)
 DATE=$(date -u +%"Y%m%dT%H%M%SZ")
-LDFLAGS_ARRAY=("github.com/stellar/kelp/cmd.version=$VERSION_STRING" "github.com/stellar/kelp/cmd.guiVersion=$GUI_VERSION" "github.com/stellar/kelp/cmd.gitBranch=$GIT_BRANCH" "github.com/stellar/kelp/cmd.gitHash=$GIT_HASH" "github.com/stellar/kelp/cmd.buildDate=$DATE" "github.com/stellar/kelp/cmd.env=$ENV")
+echo "amplitude API KEY: $AMPLITUDE_API_KEY"
```

Comment:
we don't actually want to echo this here since it's included in the LDFLAGS_ARRAY
IIRC the echo was added during development to debug and should be part of the main repo?

Reply:
Good catch, this is a leftover from our pair debugging. Removing.